src/components/HelloWorld.vue

```diff
@@ -0,0 +1,59 @@
+<template>
```

Comment:
should we remove this component? since we are only using it for unit tests AFAIK